docs/source/features/overview.rst

```diff
@@ -0,0 +1,60 @@
+Overview
+===========
```

Comment:
The underline must be at least as long as the title text (rST requirements) , but it is also good practice to make it of the same length as the title.
```suggestion
Overview
========
```